gdbgui/src/js/ProcessorsButtons.jsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+import Actions from "./Actions.js";
+import GdbApi from "./GdbApi.jsx";
+import { store } from "statorgfc";
+import constants from "./constants.js";
+
+class ProcessorsButtons extends React.Component {
+  constructor() {
+    super();
+    store.connectComponentState(this, ["processors_states"]);
+  }
+  render() {
+    let btn_class = "btn btn-default btn-sm";
```

Comment:
Bootstrap classes are being phased out in favor of [tailwind css](https://tailwindcss.com/).

Reply:
If you don’t know tailwind that’s is okay. I will update it all later.